packages/react-auth/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@carto/react-auth",
-  "version": "2.3.13",
+  "version": "2.3.11",
```

Comment:
The current version is 2.3.13 we cannot downgrade to 2.3.11 and the same for the following packages